apps/system/js/app_window.js

```diff
@@ -1703,8 +1703,19 @@
 
   AppWindow.prototype.getIconForSplash =
     function aw_getIconForSplash(manifest) {
-      var icons = this.manifest ?
-        ('icons' in this.manifest ? this.manifest.icons : null) : null;
+      var icons;
+      if (this.isPrivate) {
+        icons = {
+          '84': ' /style/icons/private_browser_84.png',
```

Comment:
This file does not exist yet. I've emailed Eric to ask about assets, maybe we should try to get those today/tomorrow then land this?